arctic/chunkstore/date_chunker.py

```diff
@@ -5,14 +5,16 @@
 
 
 class DateChunker(Chunker):
-    def to_chunks(self, df, chunk_size):
+    TYPE = 'date'
+
+    def to_chunks(self, df, chunk_size='D', **kwargs):
```

Comment:
And I found that future minute bars generally easily fit into `Y` buckets.  Maybe you've found otherwise?